src/Dev/FunctionalTest.php

```diff
@@ -449,11 +449,12 @@ public static function get_disable_themes()
         }
 
         /**
-         * @deprecated 4.2.0:5.0.0 Use ?stage=Stage in your querystring arguments instead
+         * @deprecated 4.2.0 Use ?stage=Stage in your querystring arguments instead
          * @return bool
          */
         public static function get_use_draft_site()
         {
+            Deprecation::notice('4.2.0', 'Use ?stage=Stage in your querystring arguments instead');
```

Comment:
```suggestion
         * @deprecated 4.2.0 Use ?stage=Stage in your request's querystring instead
         * @return bool
         */
        public static function get_use_draft_site()
        {
            Deprecation::notice('4.2.0', 'Use ?stage=Stage in your request's querystring instead');
```
Or keep this message, but update the other similar messages to match it. Either way is fine.

Reply:
Updated